client/src/templates/Challenges/redux/execute-challenge-saga.js

```diff
@@ -13,6 +13,8 @@ import {
   take,
   cancel
 } from 'redux-saga/effects';
+import store from 'store';
+import { Player } from 'tone';
```

Comment:
This will need to be `import()`ed, too.

Reply:
This is why I need a second set of eyes 🤦🏻